libr/util/stack.c

```diff
@@ -52,3 +52,12 @@ R_API int r_stack_is_empty(RStack *s) {
 R_API unsigned int r_stack_size(RStack *s) {
 	return (unsigned int)(s->top + 1);
```

Comment:
Shoulndt return bool?

> On 12 Aug 2016, at 02:39, Jeffrey Crowell <notifications@github.com> wrote:
> 
> In libr/util/stack.c:
> 
> > @@ -52,3 +52,12 @@ R_API int r_stack_is_empty(RStack *s) {
> >  R_API unsigned int r_stack_size(RStack *s) {
> >  	return (unsigned int)(s->top + 1);
> top is an int, this is really a size type, so :\
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>